src/server/api/organization.js

```diff
@@ -131,11 +131,25 @@ export const resolvers = {
         ? batchPolicies.split(",")
         : ["finished-replies", "vetted-texters"];
     },
-    profileFields: organization =>
-      // @todo: standardize on escaped or not once there's an interface.
-      typeof getFeatures(organization).profile_fields === "string"
-        ? JSON.parse(getFeatures(organization).profile_fields)
-        : getFeatures(organization).profile_fields || [],
+    profileFields: organization => {
+      let fields = getConfig("TEXTER_PROFILE_FIELDS", organization) || [];
```

Comment:
This is backwards incompatible -- We should test e.g. for `let fields = getConfig("TEXTER_PROFILE_FIELDS", organization) || getConfig("profile_fields", organization) || []`